lib/ansible/modules/files/synchronize.py

```diff
@@ -414,6 +419,19 @@ def main():
         rsync = module.get_bin_path(rsync, required=True)
 
     cmd = [rsync, '--delay-updates', '-F']
+    _sshpass_pipe = None
+    if rsync_password:
+        try:
+            proc = subprocess.Popen(
+                ["sshpass"], stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+            )
+            proc.communicate()
+        except OSError:
+            raise AnsibleError(
+                "to use rsync connection with passwords, you must install the sshpass program"
```

Comment:
In modules you'll need to call ```module.fail_json(msg='error message here')``` instead of raising an exception.

Reply:
thanks, fixed!